README.md

```diff
@@ -1 +1,118 @@
+
+One of the key features of Sanity implemented into this project is Live Preview, which allows you to visualise changes to your content in real-time in Sanity Studio. As the data changes in the Content Lake, our components dynamically update to reflect these changes.
+
+## Important files and folders
+
+| File(s)                          | Description                                                                                    |
+| -------------------------------- | ---------------------------------------------------------------------------------------------- |
+| `sanity.config.ts`               | Config file for Sanity Studio                                                                  |
+| `/pages/index.tsx`               | Landing page for `/` (homepage)                                                                |
+| `/pages/studio/[[...index]].tsx` | Where Sanity Studio is mounted                                                                 |
+| `/pages/api/preview.ts`          | Serverless route for triggering Preview mode (handles page slugs)                              |
+| `/pages/api/exit-preview.ts`     | Serverless route for exiting Preview mode                                                      |
+| `/sanity/env.ts`                 | Configuration for the Sanity project and dataset                                               |
+| `/sanity/schemas`                | Where Sanity Studio gets its content types from                                                |
+| `/sanity/structure.ts`           | Defines the setup for Sanity Studio's document editor view, adding an iFrame preview for pages |
+| `/sanity/lib/sanity.client.ts`   | Sanity client configured based on `env.ts`                                                     |
+| `/sanity/lib/sanity.preview.ts`  | Configures the Preview functionality                                                           |
+
+## Adding new components
+
+If you want to create more components, add additional files to the `/sanity/schemas` folder.
+
+After creating a new schema, ensure to import it and include it in the `/sanity/schemas/index.ts` file. This step is necessary for the schema to be recognised and used within Sanity Studio.
+
+Your new component must also be added to the `pageBuilder` array in the `/sanity/schema/page-schema.ts` file, like so:
+
+```
+const page = {
+  //...existing code
+  name: "pageBuilder",
+  type: "array",
+  title: "Page builder",
+  of: [
+    //...existing components
+    {
+      type: "yourNewComponent",
+    },
+  ],
+};
+```
+
+Replace `yourNewComponent` with the actual type of your new schema. This allows the new component to be used in the page building process within Sanity Studio.
+
+**TODO & DISCUSS: improve Types organisation so it makes it simpler for people to add new components and expand instructions**
```

Comment:
I've updated the code (9314e77) in the `Hero` / `CallToAction` / `PageComponents` components to import the types from the type folder. This will make it easier for users to add new components and types.